src/test/operate.test.js

```diff
@@ -0,0 +1,22 @@
+import operate from '../logic/operate';
+
+describe('Operate', () => {
+  it('Should perform addition', () => {
+    const operation = operate('6', '7', '+');
+    expect(operation).toBe('13');
+  });
+  it('Should perform subtraction', () => {
+    const operation = operate('10', '3', '-');
+    expect(operation).toBe('7');
+  });
+
+  it('Should perform modulus', () => {
+    const operation = operate('50', '10', '%');
+    expect(operation).toBe('0');
+  });
+
+  it('Should perform division', () => {
+    const operation = operate('30', '3', '÷');
+    expect(operation).toBe('10');
+  });
+});
```

Comment:
- [ ] It will be good to test for all the operations. It seems you forgot to test for multiplication in your operate.js. Kindly test for this operation since based on the requirement of the project all the operations should be tested.